akka-http-core/src/main/scala/akka/http/impl/engine/parsing/BodyPartParser.scala

```diff
@@ -289,5 +292,6 @@ private[http] object BodyPartParser {
     maxHeaderCount = 64,
```

Comment:
Gotcha, if left unaddressed please open a ticket for it - exposing config from previously hardcoded is a nice simple task  to pick up :) // cc @jrudolph